### PR TITLE
Separate open and download actions

### PR DESCRIPTION
**Pull request by hunterwerlla**
- Separate open and download actions. It still promps to download if the log stream is too big.
- Add icons and tweak text
- Fix throwing out part of the log after prompting to download
- Sprinkle on more `DumbAware`
## Screenshots
Kapture 2020-03-24 at 13 28 16

## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/DownloadLogStream.kt [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+            .builder()
+            .startFromHead(true)
+            .logGroupName(logGroup)
+            .logStreamName(logStream)
+            .endTime(startTime.toEpochMilli())
+        val getRequest = client.getLogEventsPaginator(request.build())
+        getRequest.stream().forEach {
+            indicator.checkCanceled()
+            buffer.append(it.events().buildStringFromLogsOutput())
+            index++
```
Comment (rli) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/DownloadLogStream.kt]: nit: use `forEachIndexed`

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/DownloadLogStream.kt [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+                                project,
+                                client,
+                                logGroup,
+                                logStream,
+                                buffer.toString(),
+                                request.build()
+                            )
+                        )
+                    }
+                    // Cancel this Task no matter what. If the user has agreed to download,
```
Comment (rli) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/DownloadLogStream.kt]: download file?

File: resources/resources/software/aws/toolkits/resources/localized_messages.properties [hunk 3]
```diff
@@ -571,20 +571,23 @@ cloudwatch.logs.wrap=Wrap logs
 cloudwatch.logs.tail=Tail logs
 cloudwatch.logs.show_logs_around=Show Logs Around
 cloudwatch.logs.open_in_editor=Open In Editor
+cloudwatch.logs.download_stream=Save Log Stream To File
 cloudwatch.logs.opening_in_editor=Opening Stream {0} In Editor
 cloudwatch.logs.saving_to_disk=Saving Stream {0} To Disk
-cloudwatch.logs.saving_to_disk_succeeded=Finished Saving {0}
+cloudwatch.logs.saving_to_disk_succeeded=Finished Saving Stream {0} to {1}
```
Comment (rli) [inline resources/resources/software/aws/toolkits/resources/localized_messages.properties]: finished saving log stream?

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/DownloadLogStream.kt (hunk 1 above)
Comment (hunterwerlla) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/DownloadLogStream.kt]: Apparently if we do `asSequence()` it works, changed

**Pull request merged**